config/jest/jest.config.js

```diff
@@ -67,7 +67,7 @@ module.exports = {
 	setupFilesAfterEnv: [require.resolve('./setupTests'), userSetupFile].filter(Boolean),
 	testMatch: [
 		'<rootDir>/**/__tests__/**/*.{js,jsx,ts,tsx}',
-		'<rootDir>/**/?(*.)(spec|test).{js,jsx,ts,tsx}',
+		'<rootDir>/**/?(*.)+(spec|test).[jt]s?(x)',
```

Comment:
I think the previous style can be more easily understood by developers, it's my personal opinion

Reply:
Do you mean the extension part? If someone understands the former part, (s)he will understand the changed one too.
The main purpose of this change is to fix the wrong glob pattern to the default one from jest is currently using.